packages/hooks/README.md

```diff
@@ -130,7 +134,7 @@ export function RocketInventoryList() {
 export function useMutation<TData = any, TVariables = OperationVariables>(
   mutation: DocumentNode,
   options?: MutationHookOptions<TData, TVariables>
-): MutationTuple<TData, TVariables>
+): MutationTuple<TData, TVariables>;
```

Comment:
We should give the definition of the `MutationTuple` type in this documentation, since otherwise the named type is not super helpful.

Reply:
Will do! The contents of this `README` are intended to be temporary (to help with beta testing). This `README` will be thoroughly cleaned up shortly. Thanks!